packages/story-editor/src/components/library/panes/pageTemplates/karma/defaultPageTemplates.karma.js

```diff
@@ -56,103 +56,79 @@ describe('CUJ: Page Templates: Creator can Apply a Page Template', () => {
     fixture.restore();
   });
 
-  describe('Page templates Panel', () => {
-    it('should navigate templates via keyboard', async () => {
-      // Click templates layout icon
+  const expectAllTemplatesLoaded = () => {
+    expect(fixture.screen.getAllByRole('listitem').length).toBe(
+      fixture.editor.library.pageTemplatesPane.pageTemplateButtons.length
+    );
+  };
+
+  describe('Filtering chips', () => {
+    it('should show all filtering chips', async () => {
       await fixture.editor.library.pageTemplatesTab.click();
+      await waitFor(() => expectAllTemplatesLoaded);
 
-      await waitFor(() =>
-        expect(
-          fixture.editor.library.pageTemplatesPane.pageTemplateButtons.length
-        ).toBeTruthy()
-      );
+      const filteringList = fixture.screen.getByRole('listbox', {
+        name: 'List of filtering options',
+      });
 
-      // tab to "Save current page as template" button
-      await fixture.events.keyboard.press('tab');
-      expect(fixture.editor.library.pageTemplatesPane.saveTemplateBtn).toBe(
-        document.activeElement
+      const filterOptions = within(filteringList).getAllByRole('option');
+
+      const expectedFilterOptions = [
+        'All',
+        'Cover',
+        'Section',
+        'Quote',
+        'Editorial',
+        'List',
+        'Table',
+        'Steps',
+      ];
+
+      filterOptions.map((option, index) =>
+        expect(option).toHaveTextContent(expectedFilterOptions[index])
       );
+    });
 
-      // tab to default templates dropdown button
-      await fixture.events.keyboard.press('tab');
-      expect(fixture.editor.library.pageTemplatesPane.dropDown).toBe(
-        document.activeElement
-      );
+    it('should filter templates by type', async () => {
+      await fixture.editor.library.pageTemplatesTab.click();
```

Comment:
💭 I think the current pattern here is
```suggestion
      await fixture.events.click(fixture.editor.library.pageTemplatesTab);
```

I don't know what the difference would be, but I'd go with updating this to be similar to the current pattern.

Reply:
🥰 thanks!